crates/runtime/src/dataconnector/delta_lake.rs

```diff
@@ -76,6 +76,9 @@ const PARAMETERS: &[ParameterSpec] = &[
     ParameterSpec::component("aws_endpoint")
         .description("The AWS endpoint to use for S3 storage.")
         .secret(),
+    ParameterSpec::component("aws_allow_http")
```

Comment:
I assume these params just get passed down to https://github.com/apache/arrow-rs-object-store/blob/main/src/aws/builder.rs#L405 ?

Reply:
That is correct